system-contracts/bootloader/bootloader.yul

```diff
@@ -404,7 +404,7 @@ object "Bootloader" {
             /// for the sake of simplicity we will spend 32 bytes on each
             /// of those for now. 
             function MAX_MEM_SIZE() -> ret {
```

Comment:
It seems like the space for the compressed bytecodes was not increased

Reply:
its doubled here: https://github.com/matter-labs/era-contracts/blob/ac5415a44194206d7b3a9c9048e5408aedf7d3a2/system-contracts/scripts/preprocess-bootloader.ts#L88